containersQa/testlib.bash

```diff
@@ -794,18 +794,18 @@ function checkHardcodedJdks() {
   if [ "$OTOOL_jresdk" == "jre"  ] ; then
     echo "otool jresdk settings is: $OTOOL_jresdk"
     echo "Check version based off java -version call."
-    JRE_8_VERSION='1.8.0_442-b06'
-    JRE_11_VERSION='11.0.26+4-LTS'
-    JRE_17_VERSION='17.0.14+7-LTS'
-    JRE_21_VERSION='21.0.6+7-LTS'  #temp fix until the next cpu.
+    JRE_8_VERSION='1.8.0_452-b09'
+    JRE_11_VERSION='11.0.27+6-LTS'
+    JRE_17_VERSION='17.0.15+6-LTS'
+    JRE_21_VERSION='21.0.7+6-LTS'  #temp fix until the next cpu.
     cat $(getOldJavaVersionLog)
     cat $(getOldJavaVersionLog) | grep "openjdk version"
     cat $(getOldJavaVersionLog) | grep -e "$JRE_11_VERSION" -e "$JRE_8_VERSION" -e "$JRE_17_VERSION" -e "$JRE_21_VERSION"
 
   else
     cat $(getOldMvnVersionLog)
     cat $(getOldMvnVersionLog) | grep "Java version:"
-    cat $(getOldMvnVersionLog) | grep -e "Java version: 11.0.26" -e "Java version: 1.8.0_442" -e "Java version: 17.0.14" -e "Java version: 21.0.6"
+    cat $(getOldMvnVersionLog) | grep -e "Java version: 11.0.27" -e "Java version: 1.8.0_452" -e "Java version: 17.0.15" -e "Java version: 21.0.7"
```

Comment:
on longer run, this line can use the JRE_11_VERSION values. Onyu to recalc them as `echo $JRE_11_VERSION | sed 's/[+-]+.*//' ` (untested) but I guess you got the idea....